### DISCUSSION TITLE
Document peerDependenciesMeta in package.json

### DISCUSSION THREAD

**Discussion by styfle** [npm/feedback#133] [CLI] [answered]
Looks like `peerDependenciesMeta` was added in https://github.com/npm/cli/pull/224 but never documented.

I would expect to see it here: https://docs.npmjs.com/cli/v6/configuring-npm/package-json

Also it would be great to specify the version that introduced it.

**Comment by arcanis** [comment]
For what it's worth, this field is documented on the [Yarn documentation](https://yarnpkg.com/configuration/manifest#peerDependenciesMeta).

As for npm, this flag got implemented in https://github.com/npm/cli/pull/224, which landed in 6.11.

**Comment by darcyclarke** [comment, marked as answer]
[docs.npmjs.com](https://docs.npmjs.com) should be updated for `v7` with information on `peerDependenciesMeta` [here](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta) - the `v6` docs aren't sync'd between the release & the website at the moment given that we launched a new docs experience toward the end of it's lifecycle. If you're looking for the most accurate `v6` docs, you can find them in the cli itself (ex. https://github.com/npm/cli/tree/v6/docs/content **or** `npm help-search <query>` **or** `npm help install --viewer=browser` which will launch a browser-based version locally - all of which should have the `peerDependenciesMeta` field documented as of `v6.11.0` as Maël noted)
